### PR TITLE
blockdev_commit_backing_file: Block commit "backing-file" option test

### PR DESCRIPTION
**Pull request by leidwang**
Signed-off-by: Leidong Wang <leidwang@redhat.com>
ID:1950946

### CODE REVIEW

**Comment by leidwang** [comment]
(1/1) Host_RHEL.m8.u4.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.blockdev_commit_backing_file.q35: PASS (66.42 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2021-04-19T05.16-f81095f/results.html
JOB TIME   : 67.96 s

**Comment by leidwang** [comment]
@aliang123 Could you help review this patch?  Thanks a lot.

**Comment by leidwang** [comment]
(1/1) Host_RHEL.m8.u4.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.blockdev_commit_backing_file.q35: PASS (63.25 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2021-04-21T04.38-8eb9c4a/results.html
JOB TIME   : 64.78 s

**Comment by leidwang** [comment]
(1/1) Host_RHEL.m8.u4.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.blockdev_commit_backing_file.q35: PASS (63.25 s)
RESULTS : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML : /root/avocado/job-results/job-2021-04-21T04.38-8eb9c4a/results.html
JOB TIME : 64.78 s

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.blockdev_commit_backing_file.q35: PASS (40.63 s)

Acked-by: Aihua Liang<aliang@redhat.com>

**Comment by leidwang** [comment]
@zhencliu Could you help review this patch? Thanks a lot.

File: qemu/tests/cfg/blockdev_commit_backing_file.cfg [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    image_format_sn3 = qcow2
+
+    image_name_sn4 = sn4
+    image_format_sn4 = qcow2
+
+    device_tag = "image1"
+    rebase_mode = unsafe
+    mount_point = "/tmp"
+    qemu_force_use_drive_expression = no
+    check_backing_file_cmd = 'qemu-img info %s -U'
```
Comment (zhencliu) [inline qemu/tests/cfg/blockdev_commit_backing_file.cfg:28]: what about using image_obj.info() instead?

Comment (leidwang) [reply qemu/tests/cfg/blockdev_commit_backing_file.cfg]: Updated the code,thanks a lot.

**Comment by leidwang** [comment]
(1/1) Host_RHEL.m8.u4.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.blockdev_commit_backing_file.q35: PASS (64.52 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2021-04-21T23.00-ec1b2d0/results.html
JOB TIME   : 66.05 s

**Comment by zhencliu** [comment]
LGTM

**Comment by leidwang** [comment]
@ngu-niny Could you help review this patch? Thanks a lot.

**Comment by ngu-niny** [comment]
(1/1) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.ppc64le.io-github-autotest-qemu.blockdev_commit_backing_file:  PASS (114.63 s)

Acked-by: Nini Gu <ngu@redhat.com>

**Pull request merged**